studio/components/interfaces/Organization/TeamSettings/InviteMemberButton.tsx

```diff
@@ -27,7 +27,10 @@ const InviteMemberButton: FC<Props> = ({ user, members = [], roles = [], rolesAd
 
   const [isOpen, setIsOpen] = useState(false)
   const { mutateOrgMembers } = useOrganizationDetail((slug as string) || '')
-  const canInviteMembers = checkPermissions(PermissionAction.CREATE, 'user_invites')
+
+  const canAddMembers = roles.some(({ id: role_id }) =>
```

Comment:
Optional: just curious, why was this changed from `canInviteMembers` to `canAddMembers`? I'd have thought that the former is more consistent with "invite" being used in various places.

Reply:
oh yeah good point actually - will change it back 👍